KsApi/lib/MimeType.swift

```diff
@@ -1,10 +1,13 @@
 import Foundation
 import MobileCoreServices
 
-extension NSData {
+extension Data {
   internal var imageMime: String? {
-    guard let byte: UInt8 = UnsafeBufferPointer(start: UnsafePointer(self.bytes), count: 1).first
-      else { return nil }
+
+    let start = (self as NSData).bytes.bindMemory(to: UInt8.self, capacity: self.count)
```

Comment:
@stephencelis the migrator did this and i really don't know what it's doing. if i try to get rid of the `NSData` cast it suggests to use `withUnsafeBytes`, but i'm not quite able to get the whole thing going...